### ISSUE TITLE
seems not to anonymize error.log

### ISSUE THREAD

**Issue by rseffner** [aquenos/apache2-mod-log-ipmask#3]
I tried LogDefaultIPv?Mask AND ErrorLogFormat ...%{16/48}a but client address in error.log is not anonymized

**Comment by smarsching** [comment]
You are right, so far the implementation only worked for the access log.

I made some changes that should fix this, but I don’t have the time to test them right now. Could you please checkout the code from branch `issue-3` and let me know whether this fixes the problem for you?

**Comment by rseffner** [comment]
Thanks for working on it. I build the new module (debuild is not working because of missing gpg-key) and replaced old one in /usr/lib/apache2/modules/. After stop/start apache it anonymizes access.logs but no error.log. Do I need to setup someting in MODULE.load or setup an special (Error)LogFormat?

**Comment by smarsching** [comment]
I just found out that one has to define an `ErrorLogFormat`. Otherwise, Apache never calls the format handler and instead uses an internal function for generating the log line. So, adding something like

```
ErrorLogFormat "[%t] [%m:%l] [pid\ %P:tid\ %T] [client\ %a] %E: %M"
```

on the global configuration level should do the trick (I just added this information to the README).

Can you please try this and let me know whether it fixes the problem for you?

**Comment by rseffner** [comment]
Now its working as expected. Many thanks.

**Comment by smarsching** [comment]
I just released version 1.1.0 that contains the changes. You can find a `.deb` file for Ubuntu 20.04 attached to the release. If you have to build it for another distribution, you can run `dpkg-buildpackage --no-sign` in order to make the tool work without a GPG key.
